nixos/modules/services/misc/gitlab.nix

```diff
@@ -311,6 +311,12 @@ in {
         description = "Extra configuration in config/database.yml.";
       };
 
+      redisUrl = mkOption {
+        type = types.str;
+        default = "redis://localhost:6379/";
+        description = "Redis URL for all GitLab services except gitlab-shell";
```

Comment:
What's the redis url for gitlab-shell?

Also, given this wasn't configurable before, does it make sense to default to the local unix socket?

Reply:
Well for shell it's `redis.socket = "/run/redis/gitlab.sock"` (for unix) because who needs unified ways to specify a connection anyway.

Idk about changing the default, we don't use the upstream redis module and hence I have no idea if the upstream module even creates a socket. Changing that would probably break some setups.

Reply:
If gitlab-shell connects via the local socket, and we already rely on that socket to be available and accessible, I don't see a reason why other GitLab components should not connect to it the same way.

More complicated, multi-machine setups need to deviate from the `localhost` in it anyways.

@globin, @talyz, @petabyteboy, any opinions?

Reply:
Sorry I was unclear. The socket connection is for my setup. The default is [here](https://github.com/NixOS/nixpkgs/blob/e9bf4ca80f55f33ad951e12c4eddb90d61afc1e7/nixos/modules/services/misc/gitlab.nix#L65).

Reply:
Well, in that case, fine for me.